File: code/ii-service/src/main/java/com/owlunit/service/cinema/PosterService.java
package com.owlunit.service.cinema;

import com.owlunit.model.cinema.Movie;

import java.net.URL;

/**
 * @author Ilya Pimenov
 *         Owl Proprietary
 */
public interface PosterService {

    URL getPosterUrl(Movie movie);

}

File: code/ii-service/src/main/scala/com/owlunit/service/cinema/PosterService.scala
package com.owlunit.service.cinema

import java.net.URL

/**
 * @author Anton Chebotaev
 *         Owls Proprietary
 */

trait PosterService {

  def getPosterUrl(movie: Movie):URL = new URL("http://placehold.it/200x300");

}